gaffer-as-a-service/pom.xml

```diff
@@ -35,7 +35,7 @@
     </modules>
 
     <properties>
-        <gaffer.version>1.16.0</gaffer.version>
+        <gaffer.version>1.17.0</gaffer.version>
```

Comment:
I've noticed a lot of the versions are inconsistent, is this intentional?

Reply:
Done - reverted - see below comment